internal/ast/grammar.go

```diff
@@ -56,6 +56,9 @@ func consistent(g *Grammar) (err error) {
 		defs[tok.id] = true
 	}
 	for _, prod := range g.SyntaxPart.ProdList {
+		if len(prod.Body.Symbols) == 0 {
+			return fmt.Errorf("maybe you missing the \"empty\" keyword in %q", prod)
```

Comment:
This is a great error message, that actually gives you a tip of how to fix your mistake.

Sorry to be pedantic:
```
fmt.Errorf("empty production alternative: Maybe you are missing the \"empty\" keyword in %q", prod)
```